ReportingServicesTools/Functions/CatalogItems/Set-RsSubscription.ps1

```diff
@@ -90,6 +90,7 @@ function Set-RsSubscription
     Begin
```

Comment:
For consistency sake, I would recommend you to add ApiVersion to this cmdlet also.

Reply:
Not sure it yet supports the older APIs for creating subscriptions so am going to leave this for now.